src/content/structured/get-started/releases-versioning.mdx

```diff
@@ -54,6 +54,22 @@ major.minor.patch;
 
 All packages will be published with identical version numbers.
 
-## Supporting versions
+## How we manage Beta components
 
-As the component library matures and new versions arrive, the ICDS team will actively maintain and support the previous version for up to 6 months. This is to allow the team to provide as much resource as possible on the current version. Any support for the legacy versions will only include security updates and bug fixes. No new features will be added.
+Beta components are an intermediate stage in our versioning system, between the initial "Alpha" testing phase (peer review) and the stable release. When a component reaches the Beta stage, it has passed the preliminary phase but is not yet deemed fully stable. This allows us to release components for wider testing and gather feedback from our user community while further fine-tuning their functionalities and addressing potential issues.
+
+Beta components are integrated into our normal `@ukic/` package and can be installed just like any other stable component. They are easily accessible to developers and users who wish to explore the latest developments and contribute to the improvements.
+
+It is important to be aware of a few points when working with Beta components:
+
+1. **Potential API changes:** As Beta components are still in development, there is a possibility of changes to their application programming interface (API) between major version updates. This means that updates may require manual adjustments in the codebase to accommodate the changes.
+2. **Presence of bugs:** Beta components may contain bugs or missing features that are yet to be addressed. While our team works diligently to minimise such issues, it is important to be aware of their existence.
+3. **Potential future major changes:** Since Beta components are actively evolving, there might be potential major behind-the-scenes changes planned for the future.
```

Comment:
```suggestion
3. **Potential future major changes:** Since Beta components are actively evolving, there may be major behind-the-scenes changes planned for the future.
```